src/main/java/frc/robot/subsystems/pivot/PivotIO.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.pivot;
+
+import org.littletonrobotics.junction.AutoLog;
+
+/** Add your docs here. */
+public interface PivotIO {
+  @AutoLog
+  public class PivotIOInputs {
+
+    // Motor values
+    public double velocityRPM;
+    public double currentDrawAmps;
+    public double temperatureCelsius;
+    public double motorOutputVolts;
+    public double positionRotations;
+  }
```

Comment:
same feedback as with kicker (its basically the same mech in software, just with more weight)